### DISCUSSION TITLE
Shipyard Builders '23

### DISCUSSION THREAD

**Discussion by mfahampshire** [nymtech/nym#4143] [Show and tell]
## What is this thread?
This is the discussion thread for the Shipyard Builders track hackathon submissions. 

## Where can I find submission requirements for the code repository?
You can find submission requirements [here](https://nymtech.net/developers/shipyard/guidelines.html) for your code repository. 

## How do I submit? 
Drop your submissions below as a comment including the following information:
* **Project title / name**
* **Link to repository**
* **Handle of at least one team member for contact**: Matrix preferred but Github is also fine. 
* **Challenge category**: choose one tag out of `tooling` `integrations` `miniapp`
* **Description**: One-liner
* **Demo**: Add link [if applicable]

## When do I need to submit by / whats the deadline? 
November 15th 2023 by 23:59 UTC

**Comment by gabrielclcardoso** [comment]
# Incognito Search
* Github [repo](https://github.com/TupiNymQuim/search)
* Matrix handle: [supermeia](https://matrix.to/#/@supermeia:matrix.org) [Vitor Santanna](https://matrix.to/#/@vitorsantanna:matrix.org)
* Challenge category: `miniapp`
* Description: The Anonymous Search Tool
* Demo: https://incognito-livid-nu.vercel.app/

**Comment by Pawnflake** [comment]
### ExploreNYM

- Github: N/A
- Matrix: @pawnflake:matrix.org
- Challenge category: Tooling
- Description: Delegation helper
- Demo: https://explorenym.net/

**Comment by craftdome** [comment]
### go-nym

- Github: [link](https://github.com/craftdome/go-nym) ![GitHub Repo stars](https://img.shields.io/github/stars/craftdome/go-nym?style=social)
- Matrix handle: [Evgeniy (Kronos)](https://matrix.to/#/@poweredby0:matrix.org)
- Challenge category: `tooling`
- Description: A new abstraction layer for communicating with the **nym-client** for [Golang](https://go.dev/).
- Demo: [example-app](https://github.com/craftdome/go-nym/blob/master/example/main.go)
